### PR TITLE
Pipeline to deploy MU on AWS with SUMA 4.2

### PR DESCRIPTION
**Pull request by maximenoel8**
## What does this PR change?

Adding a new pipeline to deploy SUMA 4.2 with MU on AWS

Depends on : https://github.com/uyuni-project/terracumber/pull/3

### CODE REVIEW

**Review by srbarrios — APPROVED** [review]

File: terracumber_config/tf_files/SUSEManager-4.2-AWS.tf [hunk 1]
```diff
@@ -0,0 +1,249 @@
+
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = string
+  default = "https://ci.suse.de/view/Manager/view/Manager-4.1/job/SUSEManager-4.1-AWS"
```
Comment (juliogonzalez) [inline terracumber_config/tf_files/SUSEManager-4.2-AWS.tf:5]:
```suggestion
  default = "https://ci.suse.de/view/Manager/view/Manager-4.2/job/SUSEManager-4.2-AWS"
```

File: terracumber_config/tf_files/SUSEManager-4.2-AWS.tf [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+}
+
+variable "CUCUMBER_GITREPO" {
+  type = string
+  default = "https://github.com/SUSE/spacewalk.git"
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = string
+  default = "Manager-4.1"
```
Comment (juliogonzalez) [inline terracumber_config/tf_files/SUSEManager-4.2-AWS.tf:21]:
```suggestion
  default = "Manager-4.2"
```

File: terracumber_config/tf_files/SUSEManager-4.2-AWS.tf [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = string
+  default = "/root/spacewalk/testsuite"
+}
+
+variable "MAIL_SUBJECT" {
+  type = string
+  default = "Results Manager4.1-Master-MU $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
```
Comment (juliogonzalez) [inline terracumber_config/tf_files/SUSEManager-4.2-AWS.tf:31]:
```suggestion
  default = "Results 4.2-AWS-MU $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
```

File: terracumber_config/tf_files/SUSEManager-4.2-AWS.tf [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+}
+
+variable "MAIL_TEMPLATE" {
+  type = string
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = string
+  default = "Results Manager4.1-Master-MU: Environment setup failed"
```
Comment (juliogonzalez) [inline terracumber_config/tf_files/SUSEManager-4.2-AWS.tf:41]:
```suggestion
  default = "Results 4.2-AWS-MU: Environment setup failed"
```

File: terracumber_config/tf_files/SUSEManager-4.2-AWS.tf [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+}
+
+variable "MAIL_FROM" {
+  type = string
+  default = "mnoel@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = string
+  default = "mnoel@suse.de"
```
Comment (juliogonzalez) [inline terracumber_config/tf_files/SUSEManager-4.2-AWS.tf:56]: I guess this OK for now

**Review by juliogonzalez — CHANGES_REQUESTED** [review]

**Pull request merged**